test/README.md

```diff
@@ -37,21 +37,23 @@ environment.
    [Install clusteradm CLI tool](https://open-cluster-management.io/getting-started/installation/start-the-control-plane/#install-clusteradm-cli-tool)
    for the details. Version 0.5.0 or later is required.
 
-1. Install `podman`
+1. Install `docker`
```

Comment:
I think this is a pretty big change and is claiming incompatibility with podman. I suggest that you update the commit message with example issues that you have seen with podman. If possible, also link to one or more runs on github actions where the failures have occurred.

Reply:
poadmn is considered experimental in minikube:
https://minikube.sigs.k8s.io/docs/drivers/podman/

We used it because it was a way to get rid of docker, and it seems to work, but
since we added it was removed from all environments. We kept it only in the test
env used for running the tests locally and in github. Now that it does not work
in github there is no reason to use it.

We can consider using podman again when it is officially supported and works
for us in github.

If you look in the latest github actions in this week you will find that all of
them failed in drenv tests in the same way:
- https://github.com/RamenDR/ramen/actions/runs/5348065862
- https://github.com/RamenDR/ramen/actions/runs/5347722247
- https://github.com/RamenDR/ramen/actions/runs/5347705377
- https://github.com/RamenDR/ramen/actions/runs/5347594713
- https://github.com/RamenDR/ramen/actions/runs/5347589593
- More: https://github.com/RamenDR/ramen/actions?query=is%3Afailure